homeassistant/components/motion_blinds/cover.py

```diff
@@ -339,6 +371,7 @@ async def async_stop_cover_tilt(self, **kwargs):
         """Stop the cover."""
         async with self._api_lock:
             await self.hass.async_add_executor_job(self._blind.Stop)
+        self._moving = State.Not_Moving
```

Comment:
Since there is no call to `async_request_position_till_stop`, do you need to call `self.async_write_ha_state` here so its reflected right away?

Reply:
I will add the call, but in principle a multicast callback will come in when the blind stops moving that will update the new position and does the self.async_write_ha_state.
But doing it twice to be sure in case the multicast message does not come in can not hurt I think

Reply:
done